docs/user-guide/faq.md

```diff
@@ -169,7 +174,7 @@ Check out the [Data Sets Page](https://www.cbioportal.org/datasets) for the comp
 #### Which resources are integrated for variant annotation?
 cBioPortal supports the annotation of variants from several different databases. These databases provide information about the recurrence of, or prior knowledge about, specific amino acid changes. For each variant, the number of occurrences of mutations at the same amino acid position present in the COSMIC database are reported. Furthermore, variants are annotated as “hotspots” if the amino acid positions were found to be recurrent linear hotspots, as defined by the Cancer Hotspots method ([cancerhotspots.org](https://www.cancerhotspots.org/)), or three-dimensional hotspots, as defined by 3D Hotspots ([3dhotspots.org](https://www.3dhotspots.org/)). Prior knowledge about variants, including clinical actionability information, is provided from three different sources: OncoKB ([www.oncokb.org](https://www.oncokb.org/)), CIViC ([civicdb.org](https://civicdb.org/)), as well as My Cancer Genome ([mycancergenome.org](https://www.mycancergenome.org/)). For OncoKB, exact levels of clinical actionability are displayed in cBioPortal, as defined by [the OncoKB paper](https://ascopubs.org/doi/full/10.1200/PO.17.00011).
 #### What version of the human reference genome is being used in cBioPortal?
-The [public cBioPortal](https://www.cbioportal.org) is currently using hg19/GRCh37.
+The [public cBioPortal](https://www.cbioportal.org) largely uses hg19/GRCh37. However, there are plans to incorporate more hg38/GRCh38 studies in the future; for example, the newly incorporated NCI-CRDC studies use GRCh38.
```

Comment:
... However there are studies that use the hg38/GRCh38 reference genome, including datasets sourced from the GDC through ISB-CGC BigQuery.